agent-non-conversational/app.yaml

```diff
@@ -0,0 +1,10 @@
+command: ["uv", "run", "start-server"]
+# databricks apps listen by default on port 8000
+
+env:
+  - name: MLFLOW_TRACKING_URI
+    value: "databricks"
+  - name: MLFLOW_REGISTRY_URI
+    value: "databricks-uc"
+  - name: MLFLOW_EXPERIMENT_ID
+    valueFrom: "experiment"
```

Comment:
should this just be `value` right now instead of `valueFrom` because we haven't officially rolled out experiment as a resource in prod (which is the case when we'd use valueFrom: "experiment")